rules/rfb-events.rules

```diff
@@ -0,0 +1,11 @@
+# RFB app-layer event rules.
```

Comment:
This should be squashed in the commit bringing in the events

Reply:
It should also be added to Makefile.am cf `git grep dnp3-events.rules`

Reply:
Done